lib/ansible/modules/windows/win_wait_for_process.ps1

```diff
@@ -63,6 +63,7 @@ if ($owner -and ("IncludeUserName" -notin (Get-Command -Name Get-Process).Parame
 
 Function Get-FilteredProcesses {
     [cmdletbinding()]
+    [OutputType([System.Object[]])]
```

Comment:
This is a bit useless, either we should specify the specific type of the object or just keep the `OutputType` omitted.

Reply:
In the new PR i ignored these